views/management/modifyAccount.php

```diff
@@ -12,38 +13,68 @@
 ?>
     <h1><?= Html::encode($this->title) ?></h1>
 
-    <?php $form = ActiveForm::begin([
+    <?php $formModifyAccount = ActiveForm::begin([
         'id' => 'modifyAccount-form',
         'method' => 'post',
         'layout' => 'horizontal',
         'fieldConfig' => [
             'template' => "{label}\n<div class=\"col-lg-3\">{input}</div>\n<div class=\"col-lg-8\">{error}</div>",
             'labelOptions' => ['class' => 'col-lg-1 control-label'],
-        ]
+        ],
     ]);?>
 
-    <?= $form->field($model, 'firstname')?>
+    <?= $formModifyAccount->field($formModifyAccount_model, 'firstname')?>
 
-    <?= $form->field($model, 'lastname')?>
+    <?= $formModifyAccount->field($formModifyAccount_model, 'lastname')?>
 
-    <?= $form->field($model, 'email')?>
+    <?= $formModifyAccount->field($formModifyAccount_model, 'email')?>
 
-    <?= $form->field($model, 'role')->dropDownList([
+    <?= $formModifyAccount->field($formModifyAccount_model, 'role')->dropDownList([
             'user'=>'User', 'reviewer'=>'Reviewer','publisher'=>'Publisher','admin'=>'Admin'
     ],[
         'options'=>[
             'role' => ['selected' => true]
         ]]) ?>
 
-    <?= $form->field($model,'is_active')->checkbox(); ?>
+    <?= $formModifyAccount->field($formModifyAccount_model,'is_active')->checkbox(); ?>
 
     <div class="form-group">
         <div class="col-lg-offset-1 col-lg-11">
-            <?= Html::submitButton('Submit', ['class' => 'btn btn-primary', 'name' => 'modify-button']) ?>
+            <?= Html::submitButton('Modify Account', [
+                    'class' => 'btn btn-primary', 'name' => 'modify-account-button'
+            ]) ?>
         </div>
     </div>
 
 
-    <?php ActiveForm::end(); ?>
+    <?php ActiveForm::end();
+
+    if (in_array($formModifyAccount_model->role, ['publisher','admin'])) {
```

Comment:
Why do you want to display the social media permissions form only if the edited user is a publisher or an admin ? I think that you want to make the very opposite and you must add "!" before "in_array". In this case it means that an admin cannot remove the publisher status of a user, it's not really what we need. You have to reconsider your approach